mono/mini/mini-runtime.c

```diff
@@ -3873,6 +3874,20 @@ mini_get_interp_callbacks (void)
 	return &interp_cbs;
 }
 
+static MonoDebuggerCallbacks dbg_cbs;
+
+void
+mini_install_dbg_callbacks (MonoDebuggerCallbacks *cbs)
+{
+	memcpy (&dbg_cbs, cbs, sizeof (MonoDebuggerCallbacks));
+}
```

Comment:
this should have an API version check, for example see `mono_install_icall_table_callbacks ()`